config/kibana.yml

```diff
@@ -110,4 +110,4 @@
 #ops.interval: 5000
 
 # Specifies locale to be used for all localizable strings, dates and number formats.
-#i18n.locale: "en"
+#i18n.locale: "en"
```

Comment:
revert this file, here we delete one more line break

Reply:
@zfy0701 I think we're good here now.